app/views/shared/views/_show_topology.html.haml

```diff
@@ -0,0 +1,77 @@
+- tooltipOptions = {"tooltip-placement" => "bottom", "tooltip" => "{{legendTooltip}}"}
```

Comment:
hmm... well it needs to be shared somewhere. 
I guess i could leave it under `views/container_topology/` and just direct `ems_container` to there.